python/packages/ai/teams/ai/planner/azure_openai_planner_options.py

```diff
@@ -15,8 +15,8 @@ def __init__(
         self,
         api_key: str,
         default_model: str,
-        prompt_folder: str,
         endpoint: str,
+        prompt_folder: str = "prompts",
```

Comment:
Why we need to give it a default value? The folder structure is totally defined by users so we may not make any assumptions to it.

Reply:
This just provides a suggested default that we use in our samples, I think its useful to cut down on boilerplate as much as possible and devs can always pass in an override in they want to use there own structure. When possible I like to provide default input values that can act as useful examples or for devs that might just want a simple default implementation.

Reply:
Thank you, does it make sense to make this parameter as keyword only parameter? If yes, can you move it after `*`?